algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -654,6 +693,23 @@ public Query setOptionalWords(List<String> words) {
     }
 
     /**
+     * Set the list of words that should be considered as optional when found in
+     * the query.
+     *
+     * @param words
+     *            The list of optional words.
+     */
+    public Query setOptionalWords(String... words) {
+        StringBuilder builder = new StringBuilder();
+        for (String word : words) {
+            builder.append(word);
+            builder.append(",");
+        }
+        this.optionalWords = builder.toString();
+        return this;
+    }
```

Comment:
What about just calling `setOptionalWords(Arrays.asList(words))` to avoid code duplication; no perf issue here.